src/PageCache.php

```diff
@@ -661,4 +707,41 @@ public function getStrategy()
     {
         return $this->strategy;
     }
+
+    public function clearCache()
```

Comment:
Please remove this. We prefer cleaning cache, by manually removing all dirs/files in cache storage directory.